kinto_attachment/__init__.py

```diff
@@ -23,10 +23,20 @@ def includeme(config):
 
     # Advertise public setting.
     config.registry.public_settings.add('attachment.base_url')
+    config.registry.public_settings.add('attachment.extra.base_url')
```

Comment:
Don't you think it could be a bit confusing if the name is the same? 
Try to add some documentation in the readme to see how it comes out :)